scripts/valhalla_build_elevation

```diff
@@ -2,27 +2,29 @@
 
 import argparse
 import os
-from collections import namedtuple
-from functools import partial
 import gzip
 import json
 import logging
+import sys
+
+from collections import namedtuple
+from enum import Enum, unique
+from functools import partial
```

Comment:
messing with our carefully ordered imports huh;) (though just seeing that `os` escaped) can we revert back to what it was before and just add enum alphabetically?

Reply:
haha ;) On my own projects I usually just run `black` so I don't have to think about it and usually from imports are in different blocks, but I'll just revert this and throw in enum alphabetically since this isn't my codebase :P

Reply:
yeah, would love to do that here too but installation blabla.. really, smth like `pre-commit` which runs `format.sh` would make SO much sense for valhalla as a whole (but also to run black/flake8 on python code), thousands of pointless CI hours of formatting commits saved. think I'll tinker with a shell script to do that soonish!

> usually from imports are in different blocks

interesting, never noticed! thought it's just std, installed and local modules being separated

Reply:
Yeah I didn't actually run it here, but in previous pipelines I'd set up with Python code I just run black in CI and fail if it detects any changes. Might be something to reduce cognitive overhead in the future.